packages/tocco-ui/src/EditableValue/typeEditors/DurationEdit.js

```diff
@@ -95,6 +96,9 @@ class DurationEdit extends React.Component {
     this.props.onChange(calculateMilliseconds(hoursValue, minutesValue))
   }
 
+  handleOnBlur = () => this.setState({focus: (this.state.hours >= 1 || this.state.minutes >= 1)})
```

Comment:
0 0 
This display without labels can be confusing. Are you sure you want to hide the labels for this case?

Reply:
cyril is right. 0 is a valid value.

Reply:
state.hours and state.minutes can be integer or string. Hence I prefer `this.setState({focus: (this.state.hours.length >= 1 || this.state.minutes.length >= 1)})` over `this.setState({focus: (parseInt(this.state.hours) >= 0 || parseInt(this.state.minutes) >= 0)})`.